### ISSUE TITLE
README.md should include the explanation of --cask option change.

### ISSUE THREAD

**Issue by leochoo** [Homebrew/homebrew-cask#114354]
### Provide a detailed description of the proposed feature

**TL:DR;**
Please include the usage of `brew install [packagename] --cask` in the README and USAGE.

 ------

After running into, `Error: Unknown command: cask`, I came to this [README](https://github.com/Homebrew/homebrew-cask) for a solution.
However, to my surprise, it had absolutely zero information about the `--cask` option for `brew install`.

I opened [ USAGE.md ](https://github.com/Homebrew/homebrew-cask/blob/master/USAGE.md),
and it says I should simply do `brew install`.
But since I was trying to install this cask  [Kawa](https://formulae.brew.sh/cask/kawa), which has the same name as totally different formula [Kawa](https://formulae.brew.sh/formula/kawa), this instruction was not helpful.

As my last resort, I had to google it and find my answer on [SO](https://stackoverflow.com/questions/30413621/homebrew-cask-option-not-recognized))

Overall, this was a negative user experience as I could not find the necesary information in the official docs.

I believe the usage of `brew install [] --cask` must be included in README or USAGE, as installation is the most important feature of Homebrew.

If I'm being silly and I misread something in the README or USAGE, please let me know.
But otherwise, I think it is a must to include this information.

### What is the motivation for the feature?

For the better user experience of Homebrew.

### Example use case

"Oh no, `brew cask install` throws an error - what should I do?"

"Aha! README has the answer!"

**Comment by neutric** [comment]
The two documents you mention do indeed not talk about `--cask` and could likely be improved in that regard.

Regarding Stack Overflow as last resort: 

`man brew` [says](https://docs.brew.sh/Manpage#install-options-formulacask-) `--cask Treat all named arguments as casks.` and 

`brew install kawa` warns you `Warning: Treating kawa as a formula. For the cask, use homebrew/cask/kawa`.

`brew install homebrew/cask/kawa` is thus another way of achieving what you set out to do.

**Comment by leochoo** [comment]
Thanks for letting me know of other ways to find the relevant information. As a junior developer, I do not know all the skills experienced devs have. Thus, I came to README first.

**Comment by SMillerDev** [comment]
Generally `man <tool>` is the easiest way to find how to use a tool.

**Comment by leochoo** [comment]
@miccal Is there no plan on improving the docs? Yes it is there on `man brew` but it is easy to miss and it seems unintuitive that this usage is explained within the doc for cask.

**Comment by carlocab** [comment]
We're happy to review pull requests that improve the `README`. Making it a small, simple change maximises the likelihood of any proposed changes being merged.
